server.rb

```diff
@@ -85,7 +85,7 @@ def initialize(data)
     @data = data
   end
 
-  def respond(params, &_)
+  def respond(params)
```

Comment:
As per @lbarasti 's previous suggestion, we don't ever use this block by name, only yield, [so we can pass it implicitly](http://mudge.name/2011/01/26/passing-blocks-in-ruby-without-block.html).